### ISSUE TITLE
Weapon Properties - Versatile

### ISSUE THREAD

**Issue by Meerling** [openlegend/core-rules#41]
Versatile – The weapon can be wielded either one-handed or two-handed. The wielder can freely switch between the two modes and has all of the benefits and restrictions of each while using the weapon in that mode. 

It can be interpreted such that when wielding such a weapon you'd have benefits of Advantage 1 to all attacks, while having a hand free to use another weapons or shield, and would have the restrictions of not being able to use another shield or weapon.  (I've seen people that would seriously try to take argue that in a real game.)

Please reword it to avoid that confusion. A possible suggested clarification below:

Versatile – The weapon can be wielded either one-handed or two-handed. The wielder can freely switch between the two modes and has all of the benefits and restrictions of the mode in use at that time.

**Comment by istabosz** [comment]
Good catch. Updated.
